package.json

```diff
@@ -148,7 +148,7 @@
     "into-stream": "^6.0.0",
     "jasmine": "3.5.0",
     "jasmine-core": "3.5.0",
-    "jsdom": "^24.1.1",
+    "jsdom": "^26.0.0",
```

Comment:
[`jsdom v25`](https://github.com/jsdom/jsdom/releases/tag/25.0.0) appears to improve security issues in regards to the `"dangerously"` option used by [`tasks/util/plotly_node.mjs`](https://github.com/plotly/plotly.js/blob/40f2221f0b2b1714327c5fb9bc0b1109d00aef05/tasks/util/plotly_node.mjs#L5)
This dependency bump from v24 to v26 seems good to me.